scripts/Muon/GUI/Common/fitting_tab_widget/fitting_tab_presenter.py

```diff
@@ -508,13 +429,132 @@ def clear_fit_information(self):
         self.update_fit_status_information_in_view()
         self.view.undo_fit_button.setEnabled(False)
 
-    @property
-    def start_x(self):
-        return self._start_x
+    def update_selected_workspace_guess(self):
+        if self.view.is_simul_fit():
+            self.update_fit_specifier_list()
+        if self.manual_selection_made:
+            guess_selection = self.selected_data
+            self.selected_data = guess_selection
```

Comment:
this seems circular, or am I missing something?

Reply:
Yeah, honestly not sure what the intention here was. I'll have a go at removing it.